cleancode/src/main/java/com/epam/cleancode/comments/MortgageInstallmentCalculator.java

```diff
@@ -2,38 +2,19 @@
 
 public class MortgageInstallmentCalculator {
 
-    /**
-     *
-     * @param p principal amount
-     * @param t term of mortgage in years
-     * @param r rate of interest
-     * @return monthly payment amount
-     */
-    public static double calculateMonthlyPayment(
-            int p, int t, double r) {
-
-        //cannot have negative loanAmount, term duration and rate of interest
-        if (p < 0 || t <= 0 || r < 0) {
+    public static double calculateMonthlyPayment(int principal, int mortgageTermInYears, double rateOfInterest) {
+        if (principal < 0 || mortgageTermInYears <= 0 || rateOfInterest < 0) {
             throw new InvalidInputException("Negative values are not allowed");
         }
 
-        // Convert interest rate into a decimal - eg. 6.5% = 0.065
-        r /= 100.0;
-
-        // convert term in years to term in months
-        double tim = t * 12;
-
-        //for zero interest rates
-        if(r==0)
-            return  p/tim;
+        rateOfInterest /= 100.0;
+        double mortgageTermInMonths = mortgageTermInYears * 12;
+        double monthlyRateOfInterest = rateOfInterest / 12.0;
 
-        // convert into monthly rate
-        double m = r / 12.0;
-
-        // Calculate the monthly payment
-        // The Math.pow() method is used calculate values raised to a power
-        double monthlyPayment = (p * m) / (1 - Math.pow(1 + m, -tim));
+        if (rateOfInterest == 0) {
+            return principal / mortgageTermInMonths;
+        }
 
-        return monthlyPayment;
+        return (principal * monthlyRateOfInterest) / (1 - Math.pow(1 + monthlyRateOfInterest, -mortgageTermInMonths));
```

Comment:
please simplify this operation. Eg save result of first part in one variable and second part in another variable